internal/venice-client-common/src/main/java/com/linkedin/venice/exceptions/StoreVersionNotFoundException.java

```diff
@@ -1,6 +1,6 @@
 package com.linkedin.venice.exceptions;
 
-public class StoreVersionNotFoundException extends VeniceException {
+public class StoreVersionNotFoundException extends VeniceNoStoreException {
```

Comment:
Thinking a bit more about this, I wonder if `VeniceStoreException` might be more appropriate as these extensions are not a by product of store missing rather different.

what do you think?